cmake/std/PullRequestLinuxGCC7.2.0SerialTestingSettings.cmake

```diff
@@ -0,0 +1,31 @@
+# This file contains the options needed to both run the pull request testing
+# for Trilinos for the Linux GCC 7.2.0 pull request testing builds, and to reproduce
+# the errors reported by those builds. Prior to using this this file, the
+# appropriate set of SEMS modules must be loaded and accessible through the
+# SEMS NFS mount. (See the sems/PullRequestGCC*TestingEnv.sh files.)
+
+# Usage: cmake -C PullRequestLinuxGCC7.2.0TestingSettings.cmake
+
+set (CMAKE_CXX_STANDARD "14" CACHE STRING "Set C++ standard to C++14")
+
+# Misc options typically added by CI testing mode in TriBITS
+
+# Use the below option only when submitting to the dashboard
+#set (CTEST_USE_LAUNCHERS ON CACHE BOOL "Set by default for PR testing")
+
+# Disabling fortran for this build as that is a customer use case
+set (Trilinos_ENABLE_Fortran OFF CACHE BOOL "Set by default for SERIAL PR test")
+
+set (MPI_EXEC_PRE_NUMPROCS_FLAGS "--bind-to;none" CACHE STRING "Set by default for PR testing")
+# NOTE: The above is a workaround for the problem of having threads on MPI
+# ranks bind to the same cores (see #2422).
+
+set (Trilinos_ENABLE_COMPLEX_DOUBLE ON CACHE BOOL "Set by default for PR testing to exercise complex doubles case")
+
+# Disable just one Teko sub-unit test that fails with openmpi 1.10 (#2712)
+set (Teko_DISABLE_LSCSTABALIZED_TPETRA_ALPAH_INV_D ON CACHE BOOL "Temporarily disabled in PR testing")
```

Comment:
We should be able to turn this test back on for a serial build if this is an openmpi issue.

Reply:
👍 I'll remove the line.